syntax = "proto3";
import "GSEStateFlags.proto";
import "GSEErrors.proto";

package payload;

message GSE_TO_GCS_DATA_1 {
    optional common.GSEStateFlags state_flags = 1;

    optional float transducer_1 = 2;
    optional float transducer_2 = 3;
    optional float transducer_3 = 4;

    optional float thermocouple_1 = 5;
    optional float thermocouple_2 = 6;
    optional float thermocouple_3 = 7;
    optional float thermocouple_4 = 8;

    optional common.GSEErrors error_flags = 9;
}
